### PR TITLE
Feature/visualize start stop

### LINKED ISSUE #125: Trajectory visualization: Avoid straight connections between gaps
At the moment, we simply connect all the points of a trajectory with a straight line, regardless if there was a gap in the recording or not. This can result in rather odd looking trajectory with straight  lines over long distances that look as if the user was flying from A to B.

Since we have the information when the tracking is started stored in the database, we can avoid the issue mentioned above.

Some guiding questions:

- The `start` information is stored in the database. Does the model that is displayed on the map already contain this information?

### PR DESCRIPTION
**Pull request by felixerdy**
closes #125

### CODE REVIEW

File: src/app/trajectory/map/map.page.html [hunk 1]
```diff
@@ -21,7 +21,6 @@
     (leafletMapReady)="onMapReady($event)"
     (leafletMapMoveEnd)="onMapMoved($event)"
   >
-    <div [leafletLayer]="polyline"></div>
```
Comment (felixerdy) [inline src/app/trajectory/map/map.page.html:24]: I'd be great to stay with the "layer management" we are currently using. When there are not performance issues with that, please create a `LayerGroup` (similar to `inferenceHulls`) and store all polylines in this group

File: src/app/trajectory/map/map.page.ts [hunk 2]
```diff
@@ -84,10 +84,48 @@ export class MapPage implements OnInit, OnDestroy {
     this.trajSubscription = this.trajectoryService
       .getOne(this.trajectoryType, this.trajectoryId)
       .subscribe((t) => {
-        this.polyline = new Polyline(t.coordinates, {
-          weight: 1,
+        const length = t.coordinates.length
+        const disThreshold = 30000
```
Comment (felixerdy) [inline src/app/trajectory/map/map.page.ts]: I think it's more elegant to have `disThreshold` as a class variable. It won't be reassigned every time the trajectory changes then.

File: src/app/trajectory/map/map.page.ts [hunk 3]
```diff
@@ -103,7 +141,8 @@ export class MapPage implements OnInit, OnDestroy {
           this.suppressNextMapMoveEvent = true
           this.mapBounds = this.lastLocation.getLatLng().toBounds(100)
         } else if (this.mapBounds === undefined) {
-          this.mapBounds = this.polyline.getBounds()
+          const boundPolyline = new Polyline(boundCoordinates)
+          this.mapBounds = boundPolyline.getBounds()
```
Comment (felixerdy) [inline src/app/trajectory/map/map.page.ts]: No really need to have the `boundCoordinates` variable. I guess there might be another way calculating the bounds using the polylines stored in `this.polylines` (or the LayerGroup). You can also take a look at the turfjs bbox funciton https://turfjs.org/docs/#bbox (which is already used in this project)

Comment (JuanFFranco) [reply src/app/trajectory/map/map.page.ts]: What if instead of using `LayerGroup` I use a `FeatureGroup` which allows me to use `getBounds()` function?

Comment (felixerdy) [reply src/app/trajectory/map/map.page.ts]: Yes even better I guess!

**Pull request merged**